test/Microsoft.DotNet.Tools.Tests.Utilities/TestPathUtilities.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public static class TestPathUtilities
+    {
+        public static string CreateAbsolutePath(string directoryName = null)
```

Comment:
The name here makes me think it's going to create the given directory rather than format an absolute path from the given directory name.  Perhaps `FormatAbsolutePath`?